tools/language-server/logstat/src/main.rs

```diff
@@ -285,7 +285,7 @@ fn analyze_iterations(iterations: &[Iteration], use_median: bool) -> Vec<Stats>
 
             Stats { min, max, avg, line }
         })
-        .collect_vec();
+        .collect::<Vec<_>>();
```

Comment:
Changes in LS tools caused by removal of `itertools` from prelude. That was one of the last remaining usages, and it was trivial to replace.